lib/models/collection.dart

```diff
@@ -47,6 +47,10 @@ class Collection {
     return mMdVersion > 0 && magicMetadata.visibility == kVisibilityArchive;
   }
 
+  bool isShared() {
+    return sharees.isNotEmpty || publicURLs.isNotEmpty;
```

Comment:
can either of these fields be null?

Reply:
Have added a null check. Not sure if they could be null, they can't be I think, but need more clarity on the codebase of this file.